webapp/channels/src/components/widgets/tag/tag_group.scss

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+.TagGroup {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 8px;
```

Comment:
```suggestion
```

I think this already exists in `.AlertTag` as `margin-right` property.

Reply:
done. But removed the margin-right from the alert-tag.

Reply:
@pvev Can we make this `gap: 0px 8px;`?

Because the `.AlertTag` already has a `margin-bottom: 8px`, that's causing the space between the rows to be 16px when it should be 8px.

Alternatively, we can instead remove the `margin-bottom: 8px` from the  `.AlertTag`  but I'm not sure if that'll effect anything else anywhere.